commerce_coordinator/apps/commercetools/clients.py

```diff
@@ -619,6 +620,64 @@ def update_line_item_transition_state_on_fulfillment(
             handle_commercetools_error(err, f"Unable to update LineItemState of order {order_id}", True)
             return None
 
+    def update_all_line_items_transition_state_on_fulfillment(
```

Comment:
Can we rename this to just `update_line_items_transition_state`.
Two reasons:
We are passing a list in params so it is automatically inferred that it will cater all those hence we don't need the update_all thing.
Secondly we can use this utility in generic sense and not just for updating the state due to fulfillment only hence why we need to skip on_fulfillment part and just give the sense that the utility will transition from from_state_id to new_state_key